docs/source/FAQ.rst

```diff
@@ -145,3 +145,31 @@ The issue can be fixed in two steps:
    free the PVs and you can continue with the testing but please make sure that
    you can afford to lose the data stored on the PVs you are about the free.
 
+
+**7. How can I build test-operator-index locally?**
+
+To build test-operator locally you can follow these steps inside `/test-operator`
+directory. Be sure to double-check which image name is required for each step.
+
+1. Create test-operator image
+
+.. code-block:: bash
+
+    $ make docker-build IMG=<registry>/<user>/<operator_image_name>:<tag>
+    $ make docker-push IMG=<registry>/<user>/<operator_image_name>:<tag>
+
+2. Create test-operator-bundle image
+
+.. code-block:: bash
+
+    $ make bundle IMG=<registry>/<user>/<operator_image_name>:<tag>
+    $ make bundle-build BUNDLE_IMG=<registry>/<user>/<bundle_image_name>:<tag>
+    $ make docker-push IMG=<registry>/<user>/<bundle_image_name>:<tag>
```

Comment:
This is a small thing. Can you please double check whether the `make bundle-push` really doesn't work for you? I've tested the steps locally and this worked for me:

```
make bundle-push BUNDLE_IMG=<registry>/<user>/<bundle_image_name>:<tag>
```

It would be nice if we were consistent with the openshift documentation. You were right that docker-push here works as well:) :+1: